README.md

```diff
@@ -62,138 +62,9 @@ make dev
 
 Read more about Rails Application Template in the [official Rails Guides](https://guides.rubyonrails.org/rails_application_templates.html).
 
-## How to contribute
+See the contribution guidelines in the [Wiki](../../wiki/Contribution_Guide).
```

Comment:
How about adding a heading above these links:

```
## Contribution Resources

- [Contributions Guidelines](../../wiki/Contribution_Guide).
- [Testing the template](../../wiki/Testing)
```

Reply:
Yeah. This looks much cleaner 👍🏼. Thanks. Fixed in 4415e12bdcfae4b0daa7ea1835d69fb1fe7515ab